src/main/java/transformer/ClassURLGetter.java

```diff
@@ -8,6 +8,8 @@
 
 	// TODO Refact code;
 	// TODO Javadoc;
+	// TODO new line expect Sonar to get this error;
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1135)